lib/Stan_math_code_gen.ml

```diff
@@ -73,14 +73,11 @@ let pp_block ppf (pp_body, body) = pf ppf "{@;<1 4>@[<v>%a@]@,}" pp_body body
 let pp_for_loop ppf (loopvar, lower, upper, pp_body, body) =
   pf ppf "@[<hov>for (@[<hov>size_t %s = %a;@ %s < %a;@ %s++@])" loopvar
     pp_expr lower loopvar pp_expr upper loopvar ;
-  pf ppf "@;<0 4>@[<v>%a@]@]" pp_body body
+  pf ppf "@ @;<0 4>@[<v>%a@]@]" pp_body body
```

Comment:
I'm not goin to pretend I understand this difference yet!

Reply:
Just added a `print_cut ()` effectively.